File: src/TrackInfo.java
/*
 * A class to store a Scrobble object's artist, album, and song info
 */
public class TrackInfo {
	
	private String artist;
	private String album;
	private String song;
	
	/*
	 * Creates a new TrackInfo object
	 */
	public TrackInfo(String artist, String album, String song) {
		this.artist = artist;
		this.album = album;
 		this.song = song;
	}

	// Returns the artist name
	public String getArtist() {
		return artist;
	}
	
	// Returns the album name
	public String getAlbum() {
		return album;
	}
	
	// Returns the song name
	public String getSong() {
		return song;
	}
	
	// Returns this TrackInfo represented as a string
	public String toString() {
		final String FORMAT = "%s    %s    %s";
		return String.format(FORMAT, artist, album, song);
	}
}

File: v3/track_info.py
class TrackInfo:
    '''
    A class to store a Scrobble object's artist, album, and song info
    '''

    def __init__(self, artist, album, song):
        self.__artist = artist
        self.__album = album
        self.__song = song

    def get_artist(self) -> str:
        return self.__artist
    
    def get_album(self) -> str:
        return self.__album
    
    def get_song(self) -> str:
        return self.__song
    
    def __str__(self):
        FORMAT = '{}    {}    {}'
        return FORMAT.format(self.__artist, self.__album, self.__song)
    